.gitignore

```diff
@@ -25,9 +27,6 @@ __pycache__/
 /inst/
 /llvm/
 /mingw-build/
-/nd/
-/obj/
```

Comment:
This'd be useful to keep since it's created by default with `./src/ci/docker/run.sh`

Reply:
`/obj` specifically or also `/nd`?

Reply:
Ah just `/obj`, I actually have no idea what `/nd` is!